src/views/Package/PackageDocs.tsx

```diff
@@ -42,9 +49,126 @@ const isApiPath = (path: string) => {
   return parts[parts.length - 2] === API_URL_RESOURCE;
 };
 
+/**
+ * Map from fields of `jsii-docgen.Schema` to data needed to extract values and
+ * render the right navigation sidebar. This is needed to make the information
+ * in the JSON docs play well with the existing markdown docs, but this could be
+ * refactored / dropped in the future.
+ */
+const docsSectionsMap = {
+  initializer: {
+    header: "Initializer Props",
+    getValues: (e: ClassSchema) => e.initializer?.parameters ?? [],
+    // @aws-cdk/aws-s3.Bucket.Initializer.parameter.scope => Bucket#Initializer.parameter.scope
+    getPath: (id: string) =>
+      id.split(".").slice(-4).join(".").replace(".", "#"),
```

Comment:
Can we generalize this `getPath` helper? Looks like the only difference between these functions is the slice index

Reply:
Done